docs/variable-management.md

```diff
@@ -48,3 +48,78 @@ For example, this is how variables are managed in this project:
 * additional infrastructure variables for the image-classification-tensorflow sample are defined [here](../samples/image-classification-tensorflow#cicd-in-azure-devops). They are referenced in, for example, the [train-evaluate-register DevOps pipeline](../samples/image-classification-tensorflow/devops_pipelines/03-train-evaluate-register-model.yml#L48) 
 * application variables such as the training cluster machine spec have their defaults defined in [devops_pipelines/variable-template.yml](../samples/image-classification-tensorflow/devops_pipelines/variables-template.yml) so you don't have to specify all of them in variable groups, however, based on [the order they are defined in the pipeline](../samples/image-classification-tensorflow/devops_pipelines/03-train-evaluate-register-model.yml#L46), you can overwrite their values in variables-template.yml by specifying them in the variable groups.
 * for the ML variables that need to change for experimentations, define them as [ML pipeline variables](../samples/image-classification-tensorflow/ml_service/pipelines/build_training_pipeline.py#L46). The defaults for ML variables are stored in [ml_model/parameters.json](../samples/image-classification-tensorflow/ml_model/parameters.json) so that they can run in DevOps pipelines [without having to be specified in the pipeline yamls](../samples/image-classification-tensorflow/devops_pipelines/03-train-evaluate-register-model.yml#L119).
+
+# Example 1: Image-classification-tensorflow
+
+## Infrastructure variables
+| Variable Name            | Short description                                                                      |
+| ------------------------ | ---------------------------------------------------------------------------------------|
+| ACR_SVC_CONNECTION       | Service Connection to a Docker Container Registry for build agent                      |
+| AML_COMPUTE_CLUSTER_NAME | Azure ML Compute cluster used for training                                             |
+| RESOURCE_GROUP           | Azure Resource Group where the Azure ML Workspace is located                           |
+| WORKSPACE_NAME           | Azure ML Workspace name                                                                |
+| WORKSPACE_SVC_CONNECTION | Service Connection to Azure ML Workspace                                               |
+| ACI_DEPLOYMENT_NAME      | Azure ML deployment name to Azure Container Instance                                   |
+| TEST_IMAGE_CLASSES       | a comma separated list of image classification class names of images for smoke testing |
+| TEST_IMAGE_URLS          | a comma separated URLs of images for smoke testing                                     |
```

Comment:
Can you please move these 2 variables (TEST_IMAGE_CLASSES and TEST_IMAGE_URLs) to Application variables?  Otherwise looks good.

Reply:
Yes, they are already moved to Application variables, thanks!